lib/Importer/Caffe2.cpp

```diff
@@ -470,12 +561,23 @@ void caffe2ModelLoader::loadWeights(caffe2::NetDef &net) {
       if (dict["values"]->floats_size()) {
         assert(typeName != "GivenTensorIntFill" &&
                typeName != "GivenTensorInt64Fill");
+
+        // Assume a 1D vector of length of input values.
+        if (dim.size() == 0) {
+          dim.push_back(dict["values"]->floats_size());
```

Comment:
This is 0-dims Tensor, right? Why can't we leave it as 0 dims? Have you seen cases where `dict["values"]->floats_size()` is more than 1?